README.md

```diff
@@ -0,0 +1,179 @@
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+**Hello Microverse Project** is about mastering all the tools and best practices I've learned about how to document my work in a professional way considering proper Githubflow and especially using other technical topics like linters and pull requests.
+
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://git-scm.com/">HTML and CSS</a></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Key Feature1:]**  Code Correction is done by Linter
+- **[Key Feature2:]**  Applied Correct Githubflow and documented in professional way
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- vscode.
+
+- git.
+
+- set up an index.html document.
+- set up a style.css document.
+
+### Setup
+
+Clone this repository to your desired folder:
+- Navigate to the directory where you want clone this project in your computer
+`$ cd Desktop/`
+`$ git clone https://github.com/rahimialijan/Hello-Microverse.git`
+
+### Install
+
+Install this project with this command:
+
+`$ npm install`
```

Comment:
- [ OPTIONAL]Kindly remove this since we don't to use this command in this milestone of the project `npm install` , we are running the project with `live server` :smile:

Reply:
Dear Sir, I remove the npm install line from readMe file